gist.el

```diff
@@ -186,7 +194,7 @@ With a prefix argument, makes a private paste."
 
 (defun gist-created-callback (gist)
   (let ((location (oref gist :html-url)))
-    (gist-list-reload t)
+    (gist-list-reload "" t)
```

Comment:
so to clarify a little bit, I don't mind at all if we treat nil and "" as synonyms for "current user". But then if/when that's the case, I'd prefer programmatic invocations of `gist-list-reload` to look either like
`(gist-list-reload nil t)`
or
`(gist-list-reload username t) ;; here username can be either "" or nil, I don't mind`

Using "" as a constant username is what I dislike